Exercises/checkPrime.js

```diff
@@ -0,0 +1,8 @@
+const isPrime = (number) => {
+  if (number < 2) return false;
```

Comment:
this early return case where the input number is less than 2, returning false. really looks good. I like it.